infrastructure/variables.tf

```diff
@@ -87,6 +87,10 @@ variable "common_tags" {
   type = "map"
 }
 
+variable "web_pages_appinsights_instrumentation_key" {
```

Comment:
can we have this enabled on all environments but leave it disabled for now on Prod?

Reply:
sure.